### PR TITLE
invite-store/hook: modernize the code style to match other new apps

### PR DESCRIPTION
**Pull request by tacryt-socryp**
No functionality or logic has been changed, just reorganized to be clearer and more in line with the modern style. We also no longer use the %json mark on the frontend to speak to %invite-store, instead relying on mark conversions.

Edit:

- The `%graph-create` thread should have been sending invites when it was handed an unmanaged resource with an invite policy, but it was not. It now is.

### CODE REVIEW

File: pkg/arvo/app/invite-hook.hoon
Comment (liam-fitzgerald) [inline pkg/arvo/app/invite-hook.hoon:40]: I've become kinda fond of `(team:title [our src]:bowl)` but up to you

File: pkg/arvo/app/invite-hook.hoon
Comment (liam-fitzgerald) [inline pkg/arvo/app/invite-hook.hoon:64]: Kinda think that these should maybe be moved to a `lib/invite.hoon` file, because we poke the invite-hook in a bunch of places elsewhere in userspace.

File: pkg/arvo/app/invite-store.hoon
Comment (liam-fitzgerald) [inline pkg/arvo/app/invite-store.hoon:69]: I'd rather calling `+handle-create` with `(handle-create +.action)` and then you can dodge this `?>` as well as avoiding the mildly unnecessary `act` face. Again, up to you

**Review by liam-fitzgerald — APPROVED** [review]
Looks great! Minor quibbles over style but they shouldn't block this.

File: pkg/arvo/app/invite-store.hoon
Comment (tacryt-socryp) [reply pkg/arvo/app/invite-store.hoon:69]: Good point. I straight copied the old methods, but that would be cleaner. I'll do that before we merge this.

File: pkg/arvo/sur/invite-store.hoon
Comment (liam-fitzgerald) [inline pkg/arvo/sur/invite-store.hoon:8]: based and resourcepilled

File: pkg/arvo/app/invite-store.hoon
Comment (liam-fitzgerald) [inline pkg/arvo/app/invite-store.hoon:51]: I'd make this `murn` and use `de-path-soft:res`, just in case we have some bad path formats floating around.

File: pkg/arvo/sur/invite-store.hoon
Comment (liam-fitzgerald) [inline pkg/arvo/sur/invite-store.hoon:17]: The `path` here is actually `/chat`, `/contacts`, etc. I think this should actually just be changed to `term`

File: pkg/arvo/sur/invite-store.hoon
Comment (liam-fitzgerald) [reply pkg/arvo/sur/invite-store.hoon:8]: maybe change the comment though

**Review by liam-fitzgerald — CHANGES_REQUESTED** [review]
Some issues with the conversion to `resource`. Otherwise, looks good!

File: pkg/arvo/sur/invite-store.hoon
Comment (liam-fitzgerald) [reply pkg/arvo/sur/invite-store.hoon:17]: Also these keys will need to be updated in the various `+on-init` where they're created.

File: pkg/arvo/app/invite-hook.hoon
Comment (tacryt-socryp) [reply pkg/arvo/app/invite-hook.hoon:64]: Good idea

File: pkg/arvo/app/invite-store.hoon
Comment (tacryt-socryp) [reply pkg/arvo/app/invite-store.hoon:54]: done

File: pkg/arvo/sur/invite-store.hoon
Comment (tacryt-socryp) [reply pkg/arvo/sur/invite-store.hoon]: done

**Comment by tacryt-socryp** [comment]
@liam-fitzgerald Can I get another review pass? This is ready to go and tested.

File: pkg/arvo/app/invite-hook.hoon
Comment (belisarius222) [inline pkg/arvo/app/invite-hook.hoon:49]: `%invite` should be indented once more -- or the next two args could be lifted onto this line, either way.

File: pkg/arvo/app/invite-store.hoon
Comment (belisarius222) [inline pkg/arvo/app/invite-store.hoon:85]: `path` should be indented once more, or the cases (`[%all ~]`, etc.) should be dedented.

File: pkg/arvo/app/invite-store.hoon
Comment (belisarius222) [inline pkg/arvo/app/invite-store.hoon:109]: cases should be dedented

File: pkg/arvo/ted/graph/create.hoon
Comment (belisarius222) [inline pkg/arvo/ted/graph/create.hoon:75]: `-.policy.associated.action` should be indented once more

File: pkg/arvo/app/invite-store.hoon
Comment (liam-fitzgerald) [inline pkg/arvo/app/invite-store.hoon:69]: `de-path-soft`

**Review by liam-fitzgerald — CHANGES_REQUESTED** [review]
Only blocker is `de-path-soft`. Other than that, looks good

**Review by liam-fitzgerald — APPROVED** [review]
LGTM!

**Pull request merged**
